assets/stylesheets/style.css

```diff
@@ -0,0 +1,652 @@
+}
+
+nav .nav-link:hover {
+  color: #ec5242;
+  border-radius: 5px;
+}
+
+.content {
+  background: url("../images/back.jpg") center no-repeat;
+  background-size: cover;
+  padding: 5%;
+}
+
+footer div li p {
+  font-family: "capFont", sans-serif;
+  font-size: 0.8rem;
+  color: gray;
+}
+
+.content > p {
+  font-family: "capFont", sans-serif;
+  font-size: 2rem;
+  font-style: oblique;
+  color: white;
+  max-width: 50%;
+  -webkit-text-stroke: 1px black;
+}
+
+.content h1 {
+  font-family: "headerFont", sans-serif;
+  word-wrap: break-word;
+  background: url("../images/text_back1.jpg");
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-position: top;
+  font-size: 3.5rem;
+  max-width: 60%;
+  -webkit-text-stroke: 1px black;
+}
+
+.content .card {
+  background: transparent;
+  max-width: 70%;
+  box-shadow: 1px 1px 1px 1px;
+  color: white;
+  text-shadow: -1px 1px 0 black, -1px -1px 0 black;
+  font-family: "textFont", sans-serif;
+}
+
+@media (min-width: 1000px) {
+  .content .card {
+    max-width: 45%;
+  }
+}
+
+@media (max-width: 769px) {
+  .content > p {
+    font-size: 1.85rem;
+    padding-top: 0;
+    max-width: 75%;
+  }
+
+  .content h1 {
+    font-size: 2.85rem;
+    max-width: 75%;
+  }
+
+  .content .card {
+    max-width: 75%;
+    font-size: 0.9rem;
+  }
+}
+
+@media (max-width: 550px) {
+  .content > p {
+    font-size: 1.7rem;
+    max-width: 80%;
+    padding-top: 0;
+  }
+
+  .content h1 {
+    font-size: 2.7rem;
+    max-width: 80%;
+  }
+
+  .content .card {
+    max-width: 80%;
+    font-size: 0.8rem;
+  }
+}
+
+@media (max-width: 520px) {
+  .content > * {
+    margin-inline-start: 2%;
+  }
+
+  .content > p {
+    font-size: 1.3rem;
+    max-width: 95%;
+    padding-top: 0;
+  }
+
+  .content h1 {
+    font-size: 2.3rem;
+    max-width: 95%;
+  }
+
+  .content .card {
+    max-width: 95%;
+    font-size: 0.8rem;
+  }
+}
+
+.inactive {
+  display: none;
+}
+
+footer div li {
+  font-family: "textFont", sans-serif;
+}
+
+footer div li h2 {
+  margin-top: 5%;
+  font-size: 1.2rem;
+}
+
+#programs {
+  background: url("../images/back2.jpg");
+  background-size: cover;
+  background-position: left;
+  padding: 2%;
+  max-width: 95%;
+  margin: 0 auto;
+  word-wrap: break-word;
+}
+
+#programs a {
+  display: block;
+  margin-top: 5%;
+  color: white;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+@media (max-width: 575px) {
+  #programs a {
+    display: none;
+  }
+}
+
+#programs button {
+  display: none;
+}
+
+@media (max-width: 575px) {
+  #programs button {
+    display: flex;
+    padding: 10%;
+    font-family: "capFont", sans-serif;
+    font-size: 1rem;
+    background-color: #ec5242;
+    color: black;
+    margin: 5% auto;
+    max-width: 80%;
+    border-radius: 10px;
+  }
+}
+
+footer div {
+  max-width: 90%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  padding: 2%;
+}
```

Comment:
To follow the original design, please make sure that your footer, in the mobile version of your homepage, has a white background:
![image](https://user-images.githubusercontent.com/42799579/210621480-e8b2605f-b3fe-4b0c-8b98-1026c66ec2b4.png)